cmd/hyperkube/kube-proxy.go

```diff
@@ -20,6 +20,7 @@ package main
 
 import (
 	kubeproxy "github.com/GoogleCloudPlatform/kubernetes/cmd/kube-proxy/app"
+	"github.com/GoogleCloudPlatform/kubernetes/contrib/mesos/pkg/hyperkube"
```

Comment:
These dependencies are no good. We need a better spot to put these constants so we don't link in mesoskube to hyperkube.